tests/integration/test_logs.py

```diff
@@ -1,10 +1,19 @@
 # -*- coding: utf-8 -*-
-
+import os
+import json
 import unittest
-
 from localstack.constants import APPLICATION_AMZ_JSON_1_1
 from localstack.utils.aws import aws_stack
+from localstack.utils import testutil
 from localstack.utils.common import short_uid
+from localstack.services.awslambda.lambda_api import LAMBDA_RUNTIME_PYTHON36, func_arn
+
+THIS_FOLDER = os.path.dirname(os.path.realpath(__file__))
+TEST_LAMBDA_PYTHON3 = os.path.join(THIS_FOLDER, 'lambdas', 'lambda_python3.py')
+TEST_LAMBDA_NAME_PY3 = 'test_lambda_py3'
+TEST_LAMBDA_LIBS = [
```

Comment:
nit: Can we import these constants from `test_lambda.py` instead of duplicating the code?